cg/cli/set.py

```diff
@@ -164,6 +164,9 @@ def sample(context, sex, customer, comment, downsampled_to, apptag, capture_kit,
                     echo_msg += click.style(
                         f"\nApplication tag set to {str(application_version)}.", fg='green')
 
+        context.obj['lims'].update_sample(sample_id, application=apptag)
+        click.echo(click.style(f"Set LIMS/application to {apptag}", fg='blue'))
```

Comment:
prolly good idea to add this to the `echo_msg` instead

Reply:
I think it might be better to echo it immediately since  otherwise the message might get lost if a second LIMS update fails for some reason, or what do you say?

Reply:
Good point! :+1: